ui/build.rs

```diff
@@ -9,6 +10,7 @@ use std::{
 use walkdir::WalkDir;
 
 fn main() -> Result<(), Box<dyn Error>> {
+    env::set_var("RUST_BACKTRACE", "1");
```

Comment:
any reason to set this variable while building?

Reply:
oops that was from testing a thing...